.github/workflows/webroot-deploy-development.yml

```diff
@@ -7,7 +7,7 @@ on:
   # Triggers the workflow on pushes to trunk branches involving changes to web frontend files
```

Comment:
What's the reason for the changes in this file?

Reply:
The domains were recently made permanent, allowing us to stably implement CSP. Updating the domains in this deployment pipeline was part of making it all work together.